tools/wptrunner/wptrunner/instruments.py

```diff
@@ -1,6 +1,9 @@
 import time
 import threading
-from Queue import Queue
+try:
```

Comment:
Should this use `six.moves`, something like `from six.moves.queue import Queue` ?

Reply:
Yes, it might be a better solution. Thanks!